### ISSUE TITLE
Privacy policy contradicts itself

### ISSUE THREAD

**Issue by KOLANICH** [vas3k/vas3k.club#480]
>У нас нет таргетированной рекламы, алгоритмических лент, гуглоаналитики и других сомнительных практик.

>Также на сайте происходит сбор и обработка обезличенных данных о посетителях (в т.ч. файлов «cookie») с помощью сервисов интернет-статистики (Яндекс Метрика и Гугл Аналитика и других).

**Comment by vas3k** [comment]
Я согласен, прайваси полиси нужно будет нормально переписать в следующем году. Сейчас оно по сути разрешает больше, чем мы считаем этичным. На запуске проекта всегда многое меняется, потому первичную версию полиси сделали чуть более широкой, чем было нужно, чтобы заложить возможности для маневра.

Потому да, право собирать данные Клуб получает, но делать этого не собирается. Это не противоречие :)
